.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+name: run-test
+
+# Controls when the workflow will run
+on:
+  push:
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    # Test the code on multiple python versions
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
Not a huge deal, but you probably don't need to run the build for all of these versions of Python. 3.8-3.10 are pretty standard. 3.11 is probably good, but basically brand new.